pkgs/flox-cli-tests/default.nix

```diff
@@ -177,13 +177,18 @@ writeShellScriptBin PROJECT_NAME ''
   }
   ${
     if FLOX_ACTIVATIONS_BIN == null then
-      "export FLOX_ACTIVATIONS_BIN='flox-activations';"
+      ''export FLOX_ACTIVATIONS_BIN="$(command -v flox-activations)";''
     else
       "export FLOX_ACTIVATIONS_BIN='${FLOX_ACTIVATIONS_BIN}';"
   }
   ${if FLOX_BIN == null then "export FLOX_BIN='flox';" else "export FLOX_BIN='${FLOX_BIN}';"}
   export PROCESS_COMPOSE_BIN='${process-compose}/bin/process-compose';
-  export FLOX_INTERPRETER='${flox-activation-scripts}';
+  ${
+    if flox-activation-scripts == null then
+      ''export FLOX_INTERPRETER="$PROJECT_ROOT_DIR/build/flox-activation-scripts";''
+    else
+      ''export FLOX_INTERPRETER='${flox-activation-scripts}';''
```

Comment:
nonblocking suggestions:
- do we need any of these exports if they aren't used in any of our bats files? Can we drop all of them except maybe `FLOX_BIN` and `NIX_BIN`?
- Can we get rid of the double defaulting, where there's both `flox-activations` and `FLOX_ACTIVATIONS_BIN ? "${flox-activations}/bin/flox-activations"` as arguments to this file?
- Is there a reason we need all of the `--flox-activations` arguments to this script rather than just falling back to an env var?

Reply:
There is a lot of duplication in the tests script setup
I believe most of the defaulting can be removed but I tried to keep the changes here to a minimum for this PR to keep the size and scope in check.
I'm also unsure about the impact radius of this. Also tbh, I'd prefer to just run `bats ./cli/tests` instead of going through this wrapper.